h/activity/query.py

```diff
@@ -155,16 +155,11 @@ def aggregations_for(query):
 
 
 @newrelic.agent.function_trace()
-def fetch_annotations(session, ids):
-    def load_documents(query):
-        return query.options(subqueryload(Annotation.document))
-
-    annotations = storage.fetch_ordered_annotations(
-        session, ids, query_processor=load_documents
+def _fetch_annotations(request, ids):
+    return request.find_service(AnnotationService).get_annotations_by_id(
+        ids=ids, eager_load=[Annotation.document]
     )
```

Comment:
I'm not really sure what the `newrelic` stuff is doing here, but this was a separate function in part because it gave a point to patch it out in the tests. We can do that better with `AnnotationService` mocks now.

This is a private function for this file.